autoload/ale/definition.vim

```diff
@@ -68,18 +68,26 @@ function! ale#definition#HandleLSPResponse(conn_id, response) abort
         for l:item in l:result
             if has_key(l:item, 'targetUri')
                 " LocationLink items use targetUri
-                let l:filename = ale#path#FromURI(l:item.targetUri)
+                let l:uri = l:item.targetUri
                 let l:line = l:item.targetRange.start.line + 1
                 let l:column = l:item.targetRange.start.character + 1
             else
                 " LocationLink items use uri
-                let l:filename = ale#path#FromURI(l:item.uri)
+                let l:uri = l:item.uri
                 let l:line = l:item.range.start.line + 1
                 let l:column = l:item.range.start.character + 1
             endif
 
             call ale#definition#UpdateTagStack()
-            call ale#util#Open(l:filename, l:line, l:column, l:options)
+
+            let l:uri_handler = ale#uri#GetURIHandler(l:uri)
+            if l:uri_handler is# v:null
```

Comment:
```suggestion
            let l:uri_handler = ale#uri#GetURIHandler(l:uri)

            if l:uri_handler is# v:null
```

The custom linter will complain about this. and doing this will fix it.